src/tests/Common/GenerateHWIntrinsicTests/GenerateHWIntrinsicTests_Arm.cs

```diff
@@ -3739,7 +3739,38 @@
+    ("SveVecBinOpTest.template",      new Dictionary<string, string> { ["TestName"] = "Sve_ShiftLeftLogical_uint_ulong",                                                                           ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "ShiftLeftLogical",                                                     ["RetVectorType"] = "Vector",    ["RetBaseType"] = "UInt32",  ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "UInt32", ["Op2VectorType"] = "Vector",    ["Op2BaseType"] = "UInt64",                                                                       ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetUInt32()",        ["NextValueOp2"] = "TestLibrary.Generator.GetUInt64()",                                                      ["ValidateIterResult"] = "Helpers.ShiftLeft<uint>(left[i], right[(i * sizeof(uint)) / sizeof(ulong)]) != result[i]",                    ["GetIterResult"] = "Helpers.ShiftLeft<uint>(left[i], right[(i * sizeof(uint)) / sizeof(ulong)])"}),
+
+    ("SveVecBinOpTest.template",      new Dictionary<string, string> { ["TestName"] = "Sve_ShiftRightArithmetic_sbyte_byte",                                                                       ["Isa"] = "Sve",           ["LoadIsa"] = "Sve",     ["Method"] = "ShiftRightArithmetic",                                                 ["RetVectorType"] = "Vector",    ["RetBaseType"] = "SByte",   ["Op1VectorType"] = "Vector",    ["Op1BaseType"] = "SByte",  ["Op2VectorType"] = "Vector",    ["Op2BaseType"] = "Byte",                                                                         ["LargestVectorSize"] = "64", ["NextValueOp1"] = "TestLibrary.Generator.GetSByte()",         ["NextValueOp2"] = "TestLibrary.Generator.GetByte()",                                                        ["ValidateIterResult"] = "Helpers.ShiftRight<sbyte>(left[i], (ulong)right[i]) != result[i]",                                            ["GetIterResult"] = "Helpers.ShiftRight<sbyte>(left[i], (ulong)right[i])"}),
```

Comment:
I was hoping to see something like `Helpers.ShiftRightArithmetic` for validation for these APIs?

Reply:
The generic `ShiftRight` helper will do either arithmetic or logical shift, depending on whether the first operand is signed or not since it's just doing division. I can write a `ShiftRightArithmetic` wrapper for `ShiftRight` that only takes signed types for the first operand to make the API surface clearer, though I was hoping to avoid having to write multiple overloads for `sbyte`, `short`, `int`, etc -- which would you prefer?